static/js/panels.js

```diff
@@ -91,6 +91,14 @@ export function is_timezone_inconsistent() {
     return true;
 }
 
+export function show_timezone_inconsistent_alert() {
```

Comment:
The name of the function is not consistent with what it does. It can be  `show_or_hide_timezone_update_alert` or something like that?